scripts/deployment/deployTimelockController.ts

```diff
@@ -0,0 +1,49 @@
+import * as t from "../../types/truffle-contracts/index";
+const { BN } = require("@openzeppelin/test-helpers");
+
+const TimelockController: t.TimelockControllerContract = artifacts.require("TimelockController");
+const ProxyAdmin: t.ProxyAdminContract = artifacts.require("ProxyAdmin");
+
+const ONE_MIN = new BN(60); // 60 sec
+const TEN_MINS = new BN(10).mul(ONE_MIN); // 10 mins
+const ONE_HOUR = new BN(60).mul(ONE_MIN); // 60 mins
+const FOURTY_EIGHT_HOURS = new BN(48).mul(ONE_HOUR); // 48 hours
+
+async function main() {
+    await deployTimelockController();
+    // await buildData();
+}
+async function deployTimelockController() {
+    const proposer1 = "0xebc0522e083106b5d250592f833f5213f076a45d";
+    const executor1 = "0x1d76c9f0688fce3dad2b111593477e5adc98e9e4";
+
+    const proposer2 = "0xb9f51908D332a9A6B65B6D802C84a826fa78Bb1a";
+    const executor2 = "0x7CEC1b3e1648c3F9B8fdCCDf18Ef3AbE404746D1";
```

Comment:
Why are these addresses hard coded?
Are these generated via hardhat?

Reply:
we use the original contact address from the network we are deploying TimelockController to. So these params will keep changing according to the network we are deploying.

Reply:
Alright, so maybe add something like 'Address configurabe', in the comments for these?

Reply:
not needed, as this script is only for us to use. we don't want others to use this script.